src/main/java/edu/harvard/iq/dataverse/GuestbookResponseServiceBean.java

```diff
@@ -299,7 +298,7 @@ public Long findCountAll(Long dataverseId) {
 
     private List<Object[]> convertIntegerToLong(List<Object[]> list, int index) {
         for (Object[] item : list) {
-            item[index] = new Long((Integer) item[index]);
+            item[index] = new Long((long) item[index]);
```

Comment:
Good point, @sekmiller, what does this code do? When I remove all the casts, the code just reads `item[index] = item[index]`, which is a little redundant.